Day2/BoxParser.py

```diff
@@ -0,0 +1,29 @@
+class BoxParser:
+    
+    raw_dimensions = None
+    parsed_dimensions = []
+    
+    def __init__(self, raw_dimensions):
+        self.raw_dimensions = raw_dimensions
+        self.parsed_dimensions = []
+    
+    def parse_single_box(self, box, delimiter="x"):
```

Comment:
parse single box nem int_dimensionsoket ad vissza 

single nem kell, alap angol tudasra szamithatsz, box egyes szam boxes tobbes szam
sot megint single meg ront is rajta mert ha van egy parse_box akkor nekem az a logikus h ha tobbet akarok akkor a parse_boxest keresem, nem a parse_all_boxest az all az relative es itt arra utal h a rendelkezesre allokat ami megint miert parseolnal 5ot a 198bol? alapbol vagy egyesevel vagy minden rendelkezesre allot kezelsz

az 5ot 198bol akkor jon be amikor performancen gondolkozol es mondjuk le akarsz kerni vmit a db-bol akkor lehet h a legfrissebtol indulsz 5evel mert nem fog kelleni mind de ez neked meg sok sok ideig nem jatszik